resources/styles/auth.scss

```
@@ -0,0 +1,67 @@
+@import "global.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)